_documentation/user/flags/bouncy.md

```diff
@@ -5,4 +5,18 @@ type: Bad
 summary: Tank can't stop bouncing.
 ---
 
-...
+Tanks with the Bouncy flag bounce automatically until the flag is dropped.
+
+## Tactics
+
+Bouncy automatically causes a tank to bounce each time it hits the ground. This makes landings and direction mostly uncontrolled and makes one an easier target.
+
+Countering the effects of Bouncy require fine and precise control movements and by doing so, one can turn and adjust their jump path, turning rate and be able to control their tanks general direction and path, allowing one to retain a lot of control over their tank.
+The main danger with Bouncy is the landings, which allow tanks to fire shots towards the landing path, possibly causing them to land into their shots. 
```

Comment:
Are shots ever that slow that this is room for concern?

Reply:
Intentionally jumping can also mitigate the random bounce effect, by giving you a full jump's worth of time in the air. Of course, this comes will all the risks of jumping...

Reply:
It's not about the shot speed.  He's referring to others being able to shoot where you're about to land.

Reply:
I mighty need to clarify that part, will revise it a bit for clearer wording.

Reply:
I haven't checked the code, but I'm pretty sure I've jumped when I had bouncy (which is why I said what I did). Obviously the code is the guide.

I think the challenge with this approach is that you have to hit "jump" exactly when you hit the ground (obviously, you can't jump when you are in the air), and Bouncy does make that difficult.

Reply:
@jwmelto no, the flag prevents triggering of normal jumps, it is one of its functionalities; you are at the mercy of a random height jump and being forced to the ground for 0.2 seconds each time.

Reply:
Neither of us are quite correct. The handling of the jump key is done *before* the Bouncy flag is applied (see the `wantJump` variable). Both places call `LocalPlayer::doJump` which determines if you are physically able to jump or not.

Bouncy doesn't force you to be on the ground; rather, you have 0.2 seconds between when you hit the ground and when the Bouncy bounce is applied. Manually initiating the jump in the 0.2 seconds provides a small measure of control.

Note there is a server hint that says "Self-destruct may be the easiest way to drop a Bouncy flag." All in all, it's a pretty sucky flag.

Reply:
Well, now I feel really compelled to actually try this out. I'll post practical results once I'm confident enough in seeing the results.

Reply:
@jwmelto alright, after performing a bit of testing, I'm quite confident that indeed, one *can* jump manually while holding the Bouncy flag, but such a jump is still limited in height as all other Bouncy jumps are; between 25% and 100% of a normal jump's height.

Reply:
Notes added, along with slight revision to include jump possibilities.